### PR TITLE
Make DApps search bar message more precise

### PR DESCRIPTION
**Pull request by martinmedina**
Previous DApp search bar message was "Search or enter website url" which was not correct because that bar does not allow any search to be done and it only accepts DApps URLs. An example of a DApp URL is `manager.rns.rifos.org` when a user wants to interact with. the RNS manager

This PR changes the message for "Enter the DApp website URL"  which is more precise and improves UX.

### CODE REVIEW

**Review by patogallaiovlabs — APPROVED** [review]
LGTM

File: src/common/translations/pt-BR.json [hunk 1]
```diff
@@ -278,7 +278,7 @@
     },
     "dapp": {
       "title": "DApp",
-      "search": "Faça uma busca ou insira a URL do site",
+      "search": "Insira o URL do DApp site",
```
Comment (bitcoinuser) [inline src/common/translations/pt-BR.json:281]:
```suggestion
      "search": "Insira a página URL do DApp",
```

**Review by bitcoinuser — CHANGES_REQUESTED** [review]
Minor change PT-BR

File: src/common/translations/pt.json [hunk 2]
```diff
@@ -278,7 +278,7 @@
     },
     "dapp": {
       "title": "DApp",
-      "search": "Faça uma busca ou insira a URL do site",
+      "search": "Insira o URL do DApp site",
```
Comment (patogallaiovlabs) [inline src/common/translations/pt.json:281]: Shouldn't this be as 'pt-BR'?

**Pull request merged**